admin-manual/provisioning_phone.rst

```diff
@@ -190,6 +181,23 @@ Infatti, ad ogni riavvio del telefono, quelle impostazioni non verranno mantenut
 
 Bisogna invece intervenire direttamente sull'interfaccia avanzata di |product|, modificando i template del provisioning. 
 
+I valori modificabili sono:
+
+* Lingua                                                         
+* Fuso orario
+* Formato data/ora                                        
+* Toni
+* Password utente admin                              
+* Avviso di chiamata
+* Suoneria                                                     
+* Modalità di trasferimento
+* Rubrica LDAP                                             
+* VLAN
+* Soft keys                                                     
+* Line keys
+* Exp keys                                                                                                                   
```

Comment:
What are Soft, Line and Exp keys? Could you add two or three words what they are?